omnigan/generator.py

```diff
@@ -136,11 +142,13 @@ def forward(self, x, translator="f", classifier_probs=[1, 0, 0, 0]):
         cond = None
         if self.opts.gen.t.use_spade:
             task_tensors = self.decode_tasks(z)
-            classifier_probs = torch.tensor([classifier_probs for _ in range(len(x))])
-            cond = self.get_conditioning_tensor(x, task_tensors, classifier_probs)
+            cond = get_conditioning_tensor(x, task_tensors)
         y = self.decoders["t"][translator](z, cond)
         return y
 
+    def forward(self, x, translator="f"):
```

Comment:
updated api because `forward` is not clear enough on either to feeed it a batch or a standalone image